api/src/pcapi/core/api_token/api_token.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from datetime import timedelta
+import enum
+
+from flask import current_app as app
+
+from pcapi.core.users.exceptions import InvalidToken
+from pcapi.core.users.utils import decode_jwt_token
+from pcapi.core.users.utils import encode_jwt_payload
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏La convention dans l'api est plutôt d'importer le module

```python
import pcapi.core.users.utils as users_utils

users_utils.decode_jwt_token(token)